Makefile

```diff
@@ -99,6 +99,14 @@ test: .coverage
 			-v \
 			./pfcpiface
 
+p4-constants:
+	$(info *** Generating go constants...)
+	@docker run --rm -v $(CURDIR):/app -w /app \
+		opennetworking/up4-ptf:latest python ./scripts/go_gen_p4_const.py \
```

Comment:
Is there a version available other than `:latest`?

Reply:
wait, are we using this image just for the python runtime? The script is committed here.

Reply:
I guess that this image has P4Runtime dependencies included

Reply:
Even more of an argument to write this in Go, where we have `	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"`. Keeps the P4RT versions in sync.

Reply:
I guess the reason for the python version is that we already had the script ready: https://github.com/opennetworkinglab/tassen/blob/master/util/go-gen-p4-const.py
But I agree, having it in the Go would be better!